lnwallet/chanfunding/coin_select_test.go

```diff
@@ -813,8 +842,8 @@ func TestCoinSelectUpToAmount(t *testing.T) {
 		minValue: minValue,
 		maxValue: 16 * fundingFee(feeRate, 1, false),
 
-		expectErr: "fee (0.00000192 BTC) exceeds 20% of total output " +
-			"(0.00000768 BTC)",
+		expectErr: "fee 0.00000192 BTC on total output value " +
+			"0.00000768 BTC with max fee ratio of 0.20",
```

Comment:
Nit: can we use the default ratio value here instead of hard-coding it.

Reply:
hmmm the diff would kind of get too big, would have to Sprintf the string on order to inject the "%.2f" value

we do have plenty of hard-coded values across the unit tests' expected errors, so better keep it simple?